File: src/Outros/Fibonacci.py
def fib(n):
    if n<=1:
        return n
    else:
        return fib(n-1)+ fib(n-2)

n=15
print(f"Sequencia de Fibonacci com {n} Termos ")
for i in range(n):
    print(fib(i), end=" ")

File: src/Outros/Fibonacci.java
package Outros;

public class Fibonacci {
    
     public static void main(String[] args) {
        int numero = 10;
        for (int i = 0; i < numero; i++) {
            System.out.println(fib(i) + " ");
        }

     }    
    
    public static int fib(int numero){
        if(numero<=1) return numero;
        else{
            return fib(numero-1) + fib(numero-2);
        }

    }
}
